tests/foreman/ui/test_registration.py

```diff
@@ -573,6 +582,8 @@ def test_positive_global_registration_form(
     ).create()
     iface = 'eth0'
     with session:
```

Comment:
can we use `with sat.ui_session() as session` instead of session fixture?

Reply:
I thought I covered all. Anyway I updated all tests.